### DISCUSSION TITLE
Table cells with conditioned formatting

### DISCUSSION THREAD

**Discussion by ITPLEX** [orchidsoftware/platform#1828] [General]
Hi, 

I am storing URLs in DB. How to visually show in the table cell the value exist or not (red/green color or icons)?

The idea is to show only visual information if url value exist or not (instead of whole long urls) and make the cell also clickable with link to specific stored URL.

Thanks.

**Comment by tabuna** [comment]
You can use your Blade template, for example:

```php
TD::make()->render(function($row){
    return view('...')
})
```
These strings will be passed to the closure function, and you can make a condition in the template.

**Comment by gladiiss** [comment]
Shows short url limit by 20 chars, still clickable
```
TD::make('some_heading')
    ->render(function (UrlTable $urlTable) {
        return '<a href=' . $urlTable->url . '>' . Str::limit($urlTable->url, 20) . '</a>';
    }) 
```
